temp-voyager/voyager.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://cdn.jsdelivr.net/npm/react@16/umd/react.production.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/react-dom@16/umd/react-dom.production.min.js"></script>
+
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/graphql-voyager/dist/voyager.css"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/graphql-voyager/dist/voyager.min.js"></script>
```

Comment:
Interesting usage.

Why did you choose this approach, rather than using it as a dependency / react component ? Wouldn't this make updating it more work ?

_(I'm not saying it's wrong, I'm just curious)_

Reply:
I wanted it to run as a separate script/page so my first idea was to spin up a small express server that would use the official middleware provided by this package. Unfortunately the middleware's config was limited and I couldn't get it to work with Amplify. I decided to serve this static file using express instead. It didn't seem as a bad option considering it's a temporary thing.
I thought about using their official React component but unless I wanted it to become an integral page of the cdapp (which I didn't), I would have to set up a completely separate project that I could then serve with `webpack serve`, right? Do you think there's a better approach?